server/lib/utils.js

```diff
@@ -552,3 +553,22 @@ export const computeDatesAsISOStrings = (startDate, endDate) => {
  * @returns string
  */
 export const ifStr = (condition, expression) => (condition ? expression : '');
+
+export const redactSensitiveFields = fastRedact({
+  serialize: false,
+  paths: [
+    'authorization',
+    'Authorization',
+    'AUTHORIZATION',
+    'token',
+    'accessToken',
+    'refreshToken',
+    '["Personal-Token"]',
```

Comment:
I had to come up with this funny syntax because of the dash in the variable name